INSTALL.md

```diff
@@ -23,7 +23,17 @@ For Debian-like:
 
 For Fedora-like:
 
-- dnf install gcc cmake make fuse-devel polarssl-devel ruby-devel
+- dnf install gcc cmake make fuse-devel mbedtls-devel ruby-devel rubypick
+
+Alternatively, running "dnf install dislocker fuse-dislocker" to use the
+already existing RPM packages in Fedora could be a clever idea.
+
+For RHEL-like (including CentOS Scientific Linux):
+
+- yum install gcc cmake make fuse-devel mbedtls-devel ruby-devel ruby
```

Comment:
Hehe, this is a good question which I spent quite some time yesterday with. Given the CMakefile requires /usr/bin/ruby (or a variant of), ruby-devel is unfortunately not enough. On RHEL 7, /usr/bin/ruby is shipped by 'rubypick', which is not required via the dependency chain. On RHEL 6 and 5, /usr/bin/ruby is shipped by 'ruby', which is not required via the dependency chain either. And this again leads to the point that I need to fix my suggestion even a bit, because RHEL 7 doesn't ship /usr/bin/ruby in 'ruby', but in 'rubypick' (like Fedora does). See the dependencies for CentOS 7, 6 and 5 below:

    $ rpm -qp --requires http://mirror.centos.org/centos-7/7.2.1511/os/x86_64/Packages/ruby-devel-2.0.0.598-25.el7_1.x86_64.rpm
    /usr/bin/pkg-config  
    libruby.so.2.0()(64bit)  
    rpmlib(CompressedFileNames) <= 3.0.4-1
    rpmlib(FileDigests) <= 4.6.0-1
    rpmlib(PayloadFilesHavePrefix) <= 4.0-1
    ruby(x86-64) = 2.0.0.598-25.el7_1
    rpmlib(PayloadIsXz) <= 5.2-1
    $ 

    $ rpm -qp --requires http://mirror.centos.org/centos-6/6.7/os/x86_64/Packages/ruby-devel-1.8.7.374-4.el6_6.x86_64.rpm
    libruby.so.1.8()(64bit)  
    rpmlib(CompressedFileNames) <= 3.0.4-1
    rpmlib(FileDigests) <= 4.6.0-1
    rpmlib(PayloadFilesHavePrefix) <= 4.0-1
    ruby-libs = 1.8.7.374-4.el6_6
    rpmlib(PayloadIsXz) <= 5.2-1
    $ 

    $ rpm -qp --requires http://mirror.centos.org/centos-5/5.11/os/x86_64/CentOS/ruby-devel-1.8.5-31.el5_9.x86_64.rpm
    libruby.so.1.8()(64bit)  
    rpmlib(CompressedFileNames) <= 3.0.4-1
    rpmlib(PayloadFilesHavePrefix) <= 4.0-1
    ruby-libs = 1.8.5-31.el5_9
    $